internal/store/build_result.go

```diff
@@ -105,16 +110,30 @@ type K8sBuildResult struct {
 
 	// The UIDs that we deployed to a Kubernetes cluster.
 	DeployedUIDs []types.UID
+
+	AppliedEntitiesText string
 }
 
 func (r K8sBuildResult) TargetID() model.TargetID   { return r.id }
 func (r K8sBuildResult) BuildType() model.BuildType { return model.BuildTypeK8s }
+func (r K8sBuildResult) Facets() []model.Facet {
+
+	return []model.Facet{
+		{
+			Name:  "applied yaml",
+			Value: r.AppliedEntitiesText,
+		},
+	}
+}
 
 // For kubernetes deploy targets.
-func NewK8sDeployResult(id model.TargetID, uids []types.UID) BuildResult {
+func NewK8sDeployResult(id model.TargetID, uids []types.UID, appliedEntities []k8s.K8sEntity) BuildResult {
+	appliedEntitiesYaml, _ := k8s.SerializeSpecYAML(appliedEntities)
```

Comment:
would maybe rather panic here than silently ignore the error, but 🤷‍♀

Reply:
better yet, we can stick it in the string!